frontend/__tests__/inquiry/inquiries_table.test.tsx

```diff
@@ -0,0 +1,162 @@
+// Disabling these ESLint rules for mocking the "dayjs" module
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+// Disabling to mock return value of "dayjs.tz.guess" function
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+
+import { useQuery } from "@tanstack/react-query"
+import { render, screen } from "@testing-library/react"
+import InquiriesTable from "../../src/components/Inquiries/InquiriesTable"
+import "@testing-library/jest-dom"
+
+import dayjs from "dayjs"
+
+const inquiries = [
+  {
+    id: "92bc71bf-f42c-4b56-b55d-fddaf4633550",
+    text: "How is your work-life balance?",
+    created_at: "2024-09-22T18:20:53.734830",
+  },
+  {
+    id: "736381f5-becf-4783-8f04-98250e69c0b3",
+    text: "Is communication effective?",
+    created_at: "2024-09-22T12:30:53.734830",
+  },
+  {
+    id: "df1e0d27-4f77-4d6b-b997-2b07d892db17",
+    text: "How satisfied are you with the work environment?",
+    created_at: "2024-09-21T09:20:53.734830",
+  },
+  {
+    id: "13f29f63-2793-49b5-a6a9-c26acda6651f",
+    text: "How do you feel about your current role?",
+    created_at: "2024-09-19T09:20:53.734830",
+  },
+  {
+    id: "f919b688-f9c6-49cb-87c0-4dab68a4a04c",
+    text: "How is feedback typically given and received within the team?",
+    created_at: "2024-09-22T09:20:53.734830",
+  },
+]
+
+// Mock dependencies
+jest.mock("@tanstack/react-query")
+
+describe("Inquiries Table", () => {
+  const renderComponent = () => render(<InquiriesTable />)
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it("should display empty table when there's no inquiries.", () => {
+    ;(useQuery as jest.Mock).mockReturnValue({
```

Comment:
nit:  why the semicolon at the beginning of the line?

Reply:
This must been added by my vscode prettier. Fixed!

Reply:
Actually it's not fixed, it must be something to do with pre-commit and biome setting.

Reply:
> Generally, semicolons are at the end of lines. However, in semicolon-less style, semicolons are at the beginning of lines. This rule enforces that semicolons are at the configured location.

source: https://eslint.style/rules/js/semi-style#js-semi-style

Reply:
I see that this line in biome.js is set to 
```
"semicolons": "asNeeded"
```
This seems to be intentional because it is stripping your terminating semicolons from other files.  I still have no idea why you are getting start of line semicolons.  

Something in your vscode config maybe?

Maybe @Sergei-S-V has some ideas?

Reply:
Modifying our `biome.json` javascript formatter config to
```json
"semicolons": "always"
```
removes leading semicolon, but inserts terminating semicolons to each statement.

Reply:
For my code, I think the semicolon was added to prevent ASI error.
Related Doc: https://prettier.io/docs/en/rationale.html#semicolons

My code in question:
```js
  it("should display empty table when there's no inquiries.", () => {
    ;(useQuery as jest.Mock).mockReturnValue({
      data: { data: [] },
      isPending: false,
    })
    renderComponent()
    expect(screen.getAllByRole("row").length).toBe(1) // just the header row
  })
```